src/SFA.DAS.Apim.Developer.Web/Controllers/ErrorController.cs

```diff
@@ -25,6 +25,18 @@ public ErrorController (IOptions<ExternalLinksConfiguration> configuration, Prov
         public IActionResult AccessDenied()
         {
             return View("AccessDenied", _serviceParameters.AuthenticationType == AuthenticationType.Employer ? _configuration.ManageApprenticeshipSiteUrl : _providerConfiguration.DashboardUrl + "/account");
-        }   
+        }
+        
+        [Route("404", Name = RouteNames.Error404)]
+        public IActionResult PageNotFound()
+        {
+            return View("PageNotFound", _serviceParameters.AuthenticationType == AuthenticationType.Employer ? _configuration.ManageApprenticeshipSiteUrl : _providerConfiguration.DashboardUrl + "/account");
```

Comment:
it doesn't look like these urls are used for anything on the error pages (for 404 and 500)